xml/admin_saltcluster.xml

```diff
@@ -1180,28 +1191,49 @@ OSD_ID  HOST         STATE                    PG_COUNT  REPLACE  FORCE  STARTED_
     </tip>
    </step>
    <step>
-    <para>
-     On the &smaster;, install the <package>ceph-salt</package> package if it
-     is not already installed (see <xref linkend="deploy-cephadm-cephsalt"/>),
-     and include the new &adm; in the cluster as described in
-     <xref linkend="deploy-cephadm-configure-minions"/> and
-     <xref linkend="deploy-cephadm-configure-admin"/>.
-    </para>
+     <para>
+       Install the <package>salt-master</package> package and, if applicable, the
+       <package>salt-minion</package> package on the new &smaster;.
+     </para>
    </step>
    <step>
-    <para>
-     Import the backed up cluster configuration and apply it:
-    </para>
+     <para>
+      Install &cephsalt; on the new &smaster; node:
+     </para>
 <screen>
-&prompt.smaster;ceph-salt import <replaceable>CLUSTER_CONFIG</replaceable>.json
-&prompt.smaster;ceph-salt apply
+&prompt.smaster;zypper install ceph-salt
+&prompt.smaster;systemctl restart salt-master.service
+&prompt.smaster;salt '*' saltutil.sync_all
 </screen>
-   <important>
+     <important>
+       <para>
+         Make sure to run all three commands before continuing. The commands are
+         idempotent&mdash;it does not matter if they get repeated.
```

Comment:
```suggestion
         idempotent: it does not matter if they get repeated.

Reply:
@asettle Sorry, my bad - it should be a semicolon, not a colon. I get confused about this sometimes :-/

See, e.g., https://writing.wisc.edu/handbook/grammarpunct/semicolons/

Reply:
see edited form of https://github.com/SUSE/doc-ses/pull/703#discussion_r509223348